core/iwasm/common/gc/gc_object.c

```diff
@@ -189,6 +189,16 @@ wasm_struct_obj_get_field(const WASMStructObjectRef struct_obj,
     }
 }
 
+uint32
+wasm_struct_obj_get_field_count(const WASMStructObjectRef struct_obj)
+{
+    WASMRttTypeRef rtt_type =
+        (WASMRttTypeRef)wasm_object_header((WASMObjectRef)struct_obj);
+    WASMStructType *struct_type = (WASMStructType *)rtt_type->defined_type;
+    if (struct_type->field_count > 0)
```

Comment:
Seems we don't need to add this check? Just return `struct_type->field_count`

And had better add an empty line after variable definition

Reply:
done